src/task_types/resmoke_tasks.rs

```diff
@@ -991,6 +1020,41 @@ mod tests {
         assert!(suite_2.test_list.contains(&"test_5.js".to_string()));
     }
 
+    #[test]
+    fn test_split_task_without_missing_history_should_split_tasks_equally() {
```

Comment:
I'm a bit confused, `without missing history` means that we have enough history data to split? If I'm not mistaken from the test code it seems that we don't have enough history data.

Reply:
That is a typo in the test name it should be `with missing history`.